validator/validator-main.protoascii

```diff
@@ -3523,6 +3523,21 @@ tags: {  # <amp-img>
     supported_layouts: RESPONSIVE
   }
 }
+tags: {  # <amp-layout>
+  html_format: AMP
+  html_format: AMP4ADS
+  tag_name: "AMP-LAYOUT"
+  attr_lists: "extended-amp-global"
+  spec_url: "https://www.ampproject.org/docs/reference/components/amp-layout"
+  amp_layout: {
```

Comment:
@honeybadgerdontcare is there is way to ban immediate children from being Text nodes?

Reply:
never mind, change the approach so text nodes work as well.